### PR TITLE
Create blog post for KubeVirt + MS Win workloads

### PR DESCRIPTION
**Pull request by mazzystr**
**What this PR does / why we need it**:
Blog post for migrating MS Win2K19 application servers to kubernetes and KubeVirt

**Special notes for your reviewer**:
ref Red Hat Jira [KNIECO-1864](https://issues.redhat.com/browse/KNIECO-1868)

### CODE REVIEW

File: _posts/2020-05-22-win_workload_in_k8s.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+        itemprop="thumbnail"
+        alt="vncviewer console">
+    </div>
+    > warning "NOTICE"
+    > Due to a bug with virtctl and TigerVNC a user may have to run
+    > `virtctl vnc vm`, wait for the error window to display the VNC URI,
+    > run vncviewer in another terminal and enter the ip:port manually. Project
+    > developers are working on a fix as we speak.  See GitHub
+    > [Issue #3472](https://github.com/kubevirt/kubevirt/issues/3472)
+    >
```
Comment (codificat) [inline _posts/2020-05-22-win_workload_in_k8s.md:262]: Hopefully this bug/issue is temporary - we should remember to update that step once this gets cleared

File: _posts/2020-05-22-win_workload_in_k8s.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+        alt="host web browser">
+    </div>
+    <br><br>
+  </li>
+</ol>
+
+### SUCCESS!
+
+<p>Here we have successfully demonstrated how simple it can be to migrate
+example MS Windows application and platform to kubernetes control.  For
```
Comment (codificat) [inline _posts/2020-05-22-win_workload_in_k8s.md:290]: s/example/existing/  ?

s/kubernes/Kubernetes/

File: _sass/zoom.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    -o-transform-origin: 0 0;
+    -webkit-transform: scale(1);
+    -webkit-transform-origin: 0 0;
+    transform: scale(1); /* Standard Property */
+    transform-origin: 0 0;  /* Standard Property */
+
+}
+
+.zoom:hover {
+  z-index: 1;
```
Comment (codificat) [inline _sass/zoom.scss:17]: one more thing: while looking at this in chrome I get the zoomed-in images displayed below other thumbnails.
It does look good on Firefox.

File: .yaspeller.json [hunk 4]
```diff
@@ -79,6 +82,7 @@
     "daemonset",
     "DaemonSet",
     "DaemonSets",
+    "Datacenter",
```
[the remark this answers is not in the document]
Comment (mazzystr) [reply .yaspeller.json:85]: Yea, it complained :/

File: .yaspeller.json [hunk 5]
```diff
@@ -104,6 +110,7 @@
     "Freescale",
     "frontend",
     "galleria",
+    "GB",
```
[the remark this answers is not in the document]
Comment (mazzystr) [reply .yaspeller.json:113]: I'm good with GiB :)

File: _sass/zoom.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
+.zoom {
```
[the remark this answers is not in the document]
Comment (mazzystr) [reply _sass/zoom.scss:1]: Indeed.  I didn't even notice.  I will update that.

File: _posts/2020-05-22-win_workload_in_k8s.md [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+    <div class="zoom">
+      <img
+          src="/assets/2020-05-22-win_workload_in_k8s/2-3.png"
+          width="100"
+          height="49"
+          itemprop="thumbnail"
+          alt="Verify pvc">
+    </div>
+    <br><br>
+  </li><li>Verify PersistentVolume<br>
```
[the remark this answers is not in the document]
Comment (mazzystr) [reply _posts/2020-05-22-win_workload_in_k8s.md:150]: I could go either way.   @hroyrh What do you say?  Do we care about pv data-at-rest?

File: _posts/2020-05-22-win_workload_in_k8s.md [hunk 8]
```diff
@@ -0,0 +1,291 @@
...
+      <img
+        src="/assets/2020-05-22-win_workload_in_k8s/2-4.png"
+        width="100"
+        height="47"
+        itemprop="thumbnail"
+        alt="Verify pv">
+    </div>
+    <br><br>
+  </li><li>Verify disk img data
+    <p>In a minikube kubernetes environment in order to view the file behind a
```
[the remark this answers is not in the document]
Comment (mazzystr) [reply _posts/2020-05-22-win_workload_in_k8s.md:163]: I could go either way. @hroyrh What do you say? Do we care about pv data-at-rest?

File: _posts/2020-05-22-win_workload_in_k8s.md (hunk 1 above)
Comment (mazzystr) [reply _posts/2020-05-22-win_workload_in_k8s.md:262]: Absolutely.  I created Red Hat Jira story KNIECO-2198 so we don't lose track of that.

File: _sass/zoom.scss (hunk 3 above)
Comment (mazzystr) [reply _sass/zoom.scss:17]: Confirmed and fixed in next commit.

File: .yaspeller.json [hunk 9]
```diff
@@ -134,6 +141,7 @@
     "hypervisor",
     "hypervisors",
     "Ibáñez",
+    "img",
```
[the remark this answers is not in the document]
Comment (mazzystr) [reply .yaspeller.json:144]: It has to be added due to html img tags.  yaspeller is pinging on it for some reason.

**Comment by mazzystr** [comment]
What a disaster...

**Pull request closed without merging**